core/store/src/db.rs

```diff
@@ -419,12 +437,32 @@ impl RocksDB {
             .iter()
             .map(|cf_name| ColumnFamilyDescriptor::new(cf_name, rocksdb_column_options()));
         let db = DB::open_cf_descriptors(&options, path, cf_descriptors)?;
+        #[cfg(feature = "single_thread_rocksdb")]
+        {
+            // These have to be set after open db
+            let mut env = Env::default().unwrap();
+            env.set_bottom_priority_background_threads(0);
+            env.set_high_priority_background_threads(0);
+            env.set_low_priority_background_threads(0);
+            env.set_background_threads(0);
+            println!("Disabled all background threads in rocksdb");
```

Comment:
^

Reply:
I prefer to keep this one to exolicitly know this happens on test. The above one goes into neard, probably should removed